### PR TITLE
feat: add BBS+ Tink protos

### LINKED ISSUE #2535: Create BBS+ Tink protos
part of #2295 

Create new Tink protos as we will implement the new Tink key in the framework. No change will be pushed to Tink project.

### PR DESCRIPTION
**Pull request by Baha-sk**
closes #2535

Signed-off-by: Baha Shaaban <baha.shaaban@securekey.com>

### CODE REVIEW

**Review by kdimak — APPROVED** [review]

File: proto/tink/bbs.proto [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  // BLS12_461 = 2;
+  // BLS24_479 = 3;
+  // BLS48_556 = 4;
+  // BLS48_575 = 5;
+  // BLS48_581 = 6;
+}
+
+// BBSPublicKey represents BBSVerify/BBSVerifyProof primitive.
+// key_type: type.hyperledger.org/hyperledger.aries.crypto.tink.BBSPublicKey
+message BBSPublicKey {
```
Comment (troyronda) [inline proto/tink/bbs.proto:35]: This isn't really a "BBS" key is it? isn't it a Bls12381 or similar?

File: proto/tink/bbs.proto [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+// Protos keys for BBS+ signatures/proofs.
+//
+//
+// BBS-keys represent BBSSign, BSSVerify, BBSProof and BBSVerifyProof primitives.
+
+// BBSCurveType, bls12-381 is the only supported BBS+ curve.
+enum BBSCurveType {
+  UNKNOWN_KEY_TYPE = 0;
+  BLS12_381 = 1;
```
Comment (troyronda) [inline proto/tink/bbs.proto:25]: This is not sufficient - how do you know its Bls12381G2

Comment (Baha-sk) [reply proto/tink/bbs.proto:25]: I will add GroupField to the public key..

Comment (Baha-sk) [reply proto/tink/bbs.proto:25]: added

**Review by fqutishat — APPROVED** [review]

**Pull request merged**